sites/platform/static/files/fetch/examples/relationships/redis

```diff
@@ -1,18 +1,22 @@
 {
     "username": null,
-    "scheme": "redis",
-    "service": "redis6",
     "fragment": null,
     "ip": "169.254.22.75",
-    "hostname": "7mnenhdiz7ecraovljrba6pmiy.redis6.service._.eu-3.platformsh.site",
-    "port": 6379,
     "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "redis.internal",
-    "rel": "redis",
     "path": null,
     "query": [],
     "password": null,
+    "port": 6379,
+    "host_mapped": false,
+    "service": "redis6",
+    "hostname": "7mnenhdiz7ecraovljrba6pmiy.redis6.service._.eu-3.platformsh.site",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.redis6.service._.eu-1.platformsh.site",